packages/lodestar/src/tasks/tasks/cleanUpTask.ts

```diff
@@ -0,0 +1,53 @@
+import {IBeaconChain} from "../../chain";
+import {IBeaconConfig} from "@chainsafe/lodestar-config";
+import {ILogger} from "@chainsafe/lodestar-utils";
+import {Epoch, ValidatorIndex} from "@chainsafe/lodestar-types";
+import {IBeaconDb} from "../../db";
+
+export interface ICleanUpModules {
+  chain: IBeaconChain;
+  db: IBeaconDb;
+  logger: ILogger;
+}
+
+/**
+ * Cleanup task to run per epoch or per slot.
+ */
+export class CleanUpTask {
```

Comment:
Maybe call it epoch cleanup task?

Reply:
it depends on if we want this to cleanup per epoch only or we may want to cleanup per slot later for other businesses ...

If we leave like this we'll likely have `onNewEpoch, onNewSlot` etc..

If we separate to multiple cleanup tasks then we may have EpochCleanUpTask, SlotCleanUpTask

Reply:
I changed the method name to `onNewEpoch` so that we can clean up per slot or per events later...